remote.go

```diff
@@ -18,6 +18,23 @@ import (
 	"unsafe"
 )
 
+type RemoteCreate uint
```

Comment:
could this be

```suggestion
type RemoteCreateOptionsFlag uint
```

for consistency with https://godoc.org/github.com/libgit2/git2go#BlameOptionsFlag ?  bonus points if a docstring is added.